minio/minio
# clear errors of previous failed login attempts

## Description

When the first login attempt is failed(due to incorrect secretkey)
and the next attempt is successful. Error message shown for
the previous attempts should go away.

Fixes #7514

## Motivation and Context


Stale error messages will confuse the error. If the login in succesful any 
previous error messages should be cleared.

## Regression


No

## How Has This Been Tested?



- Try login using wrong crendentials
- Login using correct credentials
- Error from failed attempt should be cleared

## Types of changes


## Checklist:
